cocos/core/animation/animation-manager.ts

```diff
@@ -55,29 +55,42 @@ export class AnimationManager extends System {
 
     public static ID = 'animation';
     private _anims = new MutableForwardIterator<AnimationState>([]);
+    private _crossFades = new MutableForwardIterator<CrossFade>([]);
```

Comment:
Since `CrossFade` may remove itself in update, we should use such constructs like `_anims`